ml-agents/mlagents/trainers/stats.py

```diff
@@ -224,12 +228,17 @@ def __init__(self, base_dir: str, clear_past_data: bool = False):
         self.summary_writers: Dict[str, SummaryWriter] = {}
         self.base_dir: str = base_dir
         self._clear_past_data = clear_past_data
+        if hidden_keys is None:
+            hidden_keys = ["Is Training", "Step"]
```

Comment:
It's probably best to set this default value on line 218. Is it conventional in Python types to pass None for an empty list?

What if I want to explicitly disable hidden keys?

Followup question, where are we injecting this?

The constructor is called in plugins/stats_writer.py's get_default_stats_writers where RunOptions (trainers/settings.py) is passed. It may make sense to add an option there.

Reply:
An empty list is a mutable data structure and the convention is not to use a mutable as argument defaults.

Right! I made the default value  an empty list [] and pass our custom hidden_keys in `plugins/stats_writer.py`: `get_default_stats_writers`